L1Trigger/Phase2L1ParticleFlow/interface/NNVtxAssoc.h

```diff
@@ -0,0 +1,44 @@
+#ifndef L1TRIGGER_PHASE2L1PARTICLEFLOWS_NNVtxAssoc_H
+#define L1TRIGGER_PHASE2L1PARTICLEFLOWS_NNVtxAssoc_H
+
+// Package:      L1Trigger/Phase2L1ParticleFlow
+// Class:        NNVtxAssoc
+// Description:  Designed to run the track to vertex associations created by the E2E NNVtx.
+//               TTTrackNetworkSelector either accepts or rejects that a PF object's (t) track is associated to a vertex (v).
+// Authors:      Kai Hong Law and Benjamin Radburn-Smith
+// Created:      February 2025
+
+#include <string>
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
```

Comment:
I think that the tensorflow::Session can be forward declared and the include moved to the .cc file.